### PR TITLE
Refactored Mainactivity.java

### LINKED ISSUE #809: [Improve code climate] Mainactivity.java
Improve code quality for MainActivity.java
Currently, the rating is "F"
Try to bring it to "B"
https://codeclimate.com/github/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/activity/MainActivity.java

### PR DESCRIPTION
**Pull request by nkarampi**
# Description

Added a new Service Layer in the application for the fragment management in the main activity. I believe adding new providers in your code, could help you to keep activities and fragments clean.

Fixes #809

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by nkarampi** [comment]
I could try to further break the complexity as some code_climate issues remain

**Comment by Swati4star** [comment]
https://codeclimate.com/github/Swati4star/Images-to-PDF/pull/821#ratings 
FragmentHandler.java has D rating. Could you work on improving that too?

**Comment by Swati4star** [comment]
Look on the issues here: https://codeclimate.com/github/Swati4star/Images-to-PDF/pull/821#issues

Try to resolve these too.

File: app/src/main/java/swati4star/createpdf/util/FragmentUtils.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+public class FragmentUtils {
+    Context mContext;
+
+    public FragmentUtils(Context mContext) {
+        this.mContext = mContext;
+    }
+
+    public String getFragmentName(Fragment fragment) {
+        String name = "set name";
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/FragmentUtils.java]: keep default name to app name

**Pull request merged**